packages/ui-kit-components/src/utils/hex-to-rgba.ts

```diff
@@ -0,0 +1,18 @@
+const hexToRgbA = (hex: string, opacity = 0.1) => {
```

Comment:
Material UI has a utility function that can do this. I think it would cover the use case. I've used it here: https://github.com/AdaptiveConsulting/ui-kit-2022/blob/19b111ab0ebdca09dc0964e9409807a452257409/packages/ui-kit-theme/src/overrides/MuiChip.ts#L97

Reply:
Thank you Ian, I've changed this with the alpha function from MUI :)